src/main/java/persistence/sql/schema/meta/EntityClassMappingMeta.java

```diff
@@ -49,7 +53,18 @@ public String fieldClause() {
 
     public List<Field> getMappingFieldList() {
         return columnMetaMap.keySet().stream()
-            .filter(field -> !ColumnMeta.isTransient(field))
+            .filter(field -> !ColumnMeta.isTransient(field) )
+            .filter(field -> !ColumnMeta.isCollection(field))
```

Comment:
```suggestion
            .filter(ColumnMeta::isNotTransient)
            .filter(ColumnMeta::isNotCollection)
```
정말 사소한 내용이지만 개인적으로 `!` 연산자 활용보다 해당 객체에 메서드 구현하여 메서드 레퍼런스로 호출하는 걸 선호합니다🙂

Reply:
제안해주신 방식이 읽기 훨씬 편하겠네요 반영하겠습니다!